### ISSUE TITLE
Feature: Check for community ban & trade ban

### ISSUE THREAD

**Issue by ghost** [ge-ku/Ban-Checker-for-Steam#50]
Its strange for me that "Ban checker for Steam" only checks in-game bans and not Steam related bans. If some people don't want this feature to be enabled by default, please add checkbox to enable check for steam account bans in extension settings.

**Comment by DraconicNEO** [comment]
I agree, this should show community and trade bans in addition to VAC bans and Game bans. It's good to be able to know these things.

**Comment by DraconicNEO** [comment]
I agree, this should show community and trade bans in addition to VAC bans and Game bans. It's good to be able to know these things.

@ge-ku Think it might be possible to add this feature to the plugin?
